packit_service/worker/handlers/distgit.py

```diff
@@ -421,6 +421,20 @@ def pre_check(self) -> bool:
                     f"Koji build configured only for '{configured_branches}'."
                 )
                 return False
+
+            # Packit should only build its own contributions
+            # We need to preserve the proven packager workflow - pushes not done by Packit
+            # should be built by those that do it, e.g. rebuilds in a side tag.
+            # prod/stg filtering happens in jobs.py -> packit_instances
+            if not (
```

Comment:
What if maintainer pushes to the PR?

Reply:
I actually don't know how pagure/fedora-messaging do it - it would still be a PR coming from Packit so I presume the event would contain Packit's credentials and would be acted upon - this is my expectation.

The long-term solution should definitely be an allowlist of FAS accounts where builds should be done automatically.

Reply:
(discussed in person)